ee/tabby-ui/app/auth/signup/components/user-register-form.tsx

```diff
@@ -62,6 +89,12 @@ export function UserAuthForm({
   buttonClass,
   ...props
 }: UserAuthFormProps) {
+  const [password, setPassword] = React.useState('')
```

Comment:
can using 'watch()' return from useForm. It should be able to solve the issue of delayed validation shown in the screen recording.

Reply:
Thanks, good point, just updated